cmd/cmd.go

```diff
@@ -73,9 +76,16 @@ func initLogging() {
 	grype.SetLogger(logWrapper)
 
 	// add a structured field to all loggers of dependencies
-	syft.SetLogger(&logger.LogrusNestedLogger{
-		Logger: logWrapper.Logger.WithField("from-lib", "syft"),
-	})
+	syftLoggerCfg := logrus.Config{
+		EnableConsole: enableConsole,
+		FileLocation:  appConfig.Log.FileLocation,
```

Comment:
Is there a possibility having 2 loggers with the same file will end up overwriting or failing to open the file?

Reply:
Removed until we can get a PR that get's both syft and grype onto the same logger